### DISCUSSION TITLE
Integration with a private server

### DISCUSSION THREAD

**Discussion by temmoai** [fattureincloud/api#322] [Q&A] [answered]
Hello, hope to be in the right place. Actually I was trying to implement the TypeScript SDK into our private backend server. 

Our needs is just to be able to create customer, save fattureInCloud ID into our Database and send them an automatic invoice when we need it. From what I understood from your documentation, its seems that the oAuth method doesnt fit with our needs.

I just need to save a key in my env variable that allow our server to access the fattureInCloud API, should I use the manual authentication? or there is another solution? 

Thank you
Matteo

**Comment by valmoz** [comment, marked as answer]
Hi @temmo93,
as explained [here](https://developers.fattureincloud.it/docs/basics/authentication#%EF%B8%8F-which-authentication-should-i-use), the Manual Token is the best solution for you if you need to manage only one Fatture in Cloud account. Just store the token properly in a backend server.

Thanks
Mauro

**Reply by valmoz** [reply]
Hi @temmo93 
can we close this discussion?

Thanks
Mauro
